### DISCUSSION TITLE
[공지] 변경된 개발 일정 공유 부탁드립니다

### DISCUSSION THREAD

**Discussion by ryan-g00** [prgrms-fe-devcourse/NFE1_2_2_MUX#48] [A. 업데이트 및 공지]
## 이번에 컴포넌트 및 파일 구조가 재구성되면서 각자 담당하는 작업의 일정이 변경되었거나 변경될 가능성이 있는 경우, 하단 커멘트로 공유해주시기 바랍니다.

[일정 변경사항 양식]
- 이슈 이름: (#이슈번호 같이 첨부)
- 변경 전 일정:
- 변경 후 일정:
- 변경 사유: (간단하게 설명)
@hjlee2778 (팀장에게 알림갈 수 있도록 태그하기)

**Comment by hjlee2778** [comment]
이슈 이름 #이슈번호: 회원 정보 수정 #8
변경 전 일정: ~ 09/28 작업
변경 후 일정: ~ 09/29 작업
변경 사유: api를 이용하여 데이터를 받아올 때 이상하게 fullName이 null로 처리되어 로그인 하였을 때 오류가 발생. 이 api를 조금 손보느라 시간이 소요될 것 같아 금일인 09/29까지 작업 후에 바로 다음 작업 진행하도록 하겠습니다.
